nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>Maybe provide an example to drive this home like a cell with "assert 1 == 2" and say when you run <code>nbdev_test_nbs</code> no error will occur.  </p><p> </p><p>Also mention that this may be useful for long running cells that you don't want to always test </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>